style.css

```diff
@@ -0,0 +1,114 @@
+body{
+    font-family: 'Montserrat', sans-serif;
+    background-color: rgb(48, 49, 70); 
+    margin: 70px;
+    margin-top: 0;
+    background-image: url(img/cover.jpg);
+    color: white;
+}
+
+.header{
+    height: 500px; 
+}
+
+.nav{
+    display: flex;
+}
+
+.nav_bar{
+    display: flex;
+    margin: 450px;
+    margin-top: 40px;
+    list-style: none;
+    font-weight: bold;
+    font-size: 14px;
+    color: white;
+}
+.nav_bar a{
+    padding: 15px;
+    text-decoration: none;
+    color: white;
+    transition: 0.2s
+}
+
+.nav_bar li{
+        margin: 5px;
+}
+
+.nav_bar a:hover{
+    color: rgb(43, 27, 116);
+}
+
+.logo{
+
+    font-size: 24px;
+}
+
+
+.main_content{
+    position: absolute;
```

Comment:
why do you set it's position to be absolute? 
If you keep the position default, you will not have to set the fixed height to the parent